README.md

```diff
@@ -134,37 +134,7 @@ The versioning in Filecoin-FFI currently follows the Lotus versioning.  For exam
 
 #### Release Process
 
-##### Cutting a development or release candidate release
-
-<details>
-  <summary>Steps to cut a development or release candidate release:</summary>
-
-1. Go to [Filecoin-FFI Releases](https://github.com/filecoin-project/filecoin-ffi/releases).
-2. Click the "Draft a new release" button in the right corner.
-3. In the "Choose a tag" dropdown, enter the desired version and click "Create new tag: vX.XX.X-rcX/dev on publish".
-4. Target the master branch.
-5. Set the previous tag to compare against, which should be the last stable release.
-6. Click the "Generate release notes" button.
-6. Check the "Set as a pre-release" checkbox.
-7. Click "Publish release" to create the release.
-
-</details>
-
-##### Cutting a definitive release
-
-<details>
-  <summary>Steps to cut a definitive release:</summary>
-
-1. Go to [Filecoin-FFI Releases](https://github.com/filecoin-project/filecoin-ffi/releases).
-2. Click the "Draft a new release" button in the right corner.
-3. In the "Choose a tag" dropdown, enter the desired version and click "Create new tag: vX.XX.X on publish".
-4. Target the release candidate you want to create a stable.
-5. Set the previous tag to compare against, which should be the last stable release (e.g., non-RC)
-6. click the "Generate release notes" button.
-6. Ensure the "Set as a latest release" checkbox **is checked**.
-7. Click "Publish release" to create the release.
-
-</details>
+Create a new entry in the [Changelog](CHANGELOG.md) for the new release. If you don't provide any description/body for the new entry, the release will be created with auto-generated release notes.
```

Comment:
```suggestion
Create a new entry in the [Changelog](CHANGELOG.md) via PR for the new release using the [keep a changelog format](https://keepachangelog.com/en/1.1.0/). 
When the PR is opened, a draft GitHub release will be created.
When the PR is merged, the GitHub release will be published.
If you don't provide any description/body for the new CHANGELOG entry, the release will be created with GitHub auto-generated release notes.
You can read more about this release flow at https://github.com/ipdxco/changelog-driven-release
```

Might just be me, but this section seemed a little too sparse.  I know the process is simple (good), but I think a bit more info is useful.

https://keepachangelog.com/en/1.1.0/ doesn't seem like the right link here given it doesn't specify the ``## [x.y.z] yyyy-mm-dd`` format I don't think.  Maybe it would be better to link to https://github.com/ipdxco/changelog-driven-release?tab=readme-ov-file ?